plugins/http_plugin/http_plugin.cpp

```diff
@@ -689,6 +732,8 @@ namespace eosio {
              "The maximum body size in bytes allowed for incoming RPC requests")
             ("http-max-bytes-in-flight-mb", bpo::value<uint32_t>()->default_value(500),
              "Maximum size in megabytes http_plugin should use for processing http requests. 503 error response when exceeded." )
+            ("http-max-in-flight-requests", bpo::value<int32_t>()->default_value(-1),
+             "Maximum number of requests http_plugin should use for processing http requests. 503 error response when exceeded." )
```

Comment:
This doc, is wrong right? We are sending error 429 not 503.

Reply:
> This doc, is wrong right? We are sending error 429 not 503.

It's the same as http-max-bytes-in-flight-mb. So we should change the dic for http-max-bytes-in-flight-mb as well?

Reply:
yes

Reply:
Make sure to update the PR description with the documentation changes.

Reply:
> Make sure to update the PR description with the documentation changes.

Updated the code. PR description is the "Change Description" above, right?

Reply:
Specifically the "Documentation Additions" above. You need to list your new option with its doc and note that the description changed on the other one.